### PR TITLE
Adds 3 simple fields to `src/index.ts` scaffold with a sample import

### PR DESCRIPTION
**Pull request by adambouchard**
Purpose

Adds 3 simple fields to scaffold with a sample import CSV (`examples/imports/myWorkbookExample.csv`) to make it easier for a new customer to deploy the default workbook and follow Dev Docs to example to try uploading data...

### CODE REVIEW

File: examples/imports/defaultWorkbook.csv [hunk 1]
```diff
@@ -0,0 +1,51 @@
+firstName,lastName,email
```
Comment (alexhollenbeck) [inline examples/imports/defaultWorkbook.csv:1]: Can we call this `myWorkbookExample.csv` to match the name of the workbook it's associated with (`MyWorkbook`)?

Comment (adambouchard) [reply examples/imports/defaultWorkbook.csv]: updated

File: src/index.ts [hunk 2]
```diff
@@ -19,7 +19,9 @@ import { YourSheet } from '/path-to-your-sheet/your-sheet.ts'
 const MySheet = new Sheet(
   'MySheet',
   {
-    exampleField: TextField(),
+    firstName: TextField(),
+    lastName: TextField(),
+    email: TextField()
```
Comment (paddymul) [inline src/index.ts:24]: I would avoid any email until we build a proper email field.  maybe age? as a NumberField instead?

File: src/index.ts [hunk 3]
```diff
@@ -19,7 +19,9 @@ import { YourSheet } from '/path-to-your-sheet/your-sheet.ts'
 const MySheet = new Sheet(
   'MySheet',
   {
-    exampleField: TextField(),
+    firstName: TextField(),
+    lastName: TextField(),
+    dob: DateField()
```
Comment (paddymul) [inline src/index.ts:24]: Please make this "age", "NumberField".  Our DateField could use some refinement before putting it right in front of users.

**Review by paddymul — APPROVED** [review]

**Pull request merged**